src/mscorlib/src/System/Resources/IResourceManager.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+
+namespace System.Resources
+{
+    /// <summary>
+    /// Private interface between CoreLib and ResourceManager so that CoreLib can access resources
```

Comment:
https://github.com/dotnet/corefx/blob/master/src/Common/src/System/SR.Core.cs indirection is ProjectN specific hack. Does this mean that we will have this hack in more assemblies and in CoreCLR as well now? It sounds pretty unfortunate.

This hack is using WinRTInterop.Callbacks. It does not exist in CoreCLR. What is it going to use in CoreCLR instead?